packages/lib/src/components/internal/SecuredFields/SFP/types.ts

```diff
@@ -11,7 +11,7 @@ export interface SFPState {
     isSfpValid?: boolean;
     autoCompleteName?: string;
     billingAddress?: AddressData;
-    hasUnsupportedCard?: boolean;
```

Comment:
Just a bit weird the naming of the variable. It made more sense when it was a `boolean` type :)